connectors-e2e-test/connectors-e2e-test-mail/src/test/java/io/camunda/connector/e2e/BaseEmailTest.java

```diff
@@ -67,7 +68,10 @@ protected static List<String> getReceivers(Message message) {
 
   protected static String getPlainTextBody(Message message) {
     try {
-      return message.getContent().toString().trim();
+      if (message.getContent() instanceof Multipart multipart) {
+        return multipart.getBodyPart(0).getContent().toString();
```

Comment:
Shouldnt we look at the content type when fetching the plain text part? Or is it guaranteed to always be the first one?